asustor.c

```diff
@@ -19,145 +19,121 @@
 #include <linux/platform_device.h>
 #include <linux/version.h>
 
-#define GPIO_IT87                                                              \
-	"asustor_gpio_it87" // use custom patched version for IT8625 support
+#define GPIO_IT87 "asustor_gpio_it87"
 #define GPIO_ICH "gpio_ich"
 #define GPIO_AS6100 "INT33FF:01"
 
-// clang-format off
+#define DISK_ACT_LED(_name)                                                    \
+	{                                                                      \
+		.name		 = _name ":green:disk",                        \
+		.default_state	 = LEDS_GPIO_DEFSTATE_ON,                      \
+		.default_trigger = "disk-activity"                             \
+	}
+#define DISK_ERR_LED(_name)                                                    \
+	{                                                                      \
+		.name	       = _name ":red:disk",                            \
+		.default_state = LEDS_GPIO_DEFSTATE_OFF                        \
+	}
 
 // ASUSTOR Leds.
 // If ledtrig-blkdev ever lands, use that instead of disk-activity:
 // https://lore.kernel.org/linux-leds/20210819025053.222710-1-arequipeno@gmail.com/
 static struct gpio_led asustor_leds[] = {
-	{ .name = "blue:power",   .default_state = LEDS_GPIO_DEFSTATE_ON },    //  0
-	{ .name = "red:power",    .default_state = LEDS_GPIO_DEFSTATE_OFF },   //  1
-	{ .name = "green:status", .default_state = LEDS_GPIO_DEFSTATE_ON },    //  2
+	{ .name		 = "power:front_panel",
+	  .default_state = LEDS_GPIO_DEFSTATE_ON }, // 1
```

Comment:
The commented indices are wrong - here it should be `// 0` not `// 1`, equivalent for all the following ones.
(I added those comments to make it easier to write the lookup tables, for the fourth argument of `GPIO_LOOKUP_IDX()`)

also, for the multi-line entries, I'd put the index comments on the name line

and maybe make those comments align in lines of similar length, like
```
	{ .name = "blue:usb", .default_state = LEDS_GPIO_DEFSTATE_OFF },  // 6
	{ .name = "green:usb", .default_state = LEDS_GPIO_DEFSTATE_OFF }, // 7
	{ .name = "blue:lan", .default_state = LEDS_GPIO_DEFSTATE_ON },   // 8
	DISK_ACT_LED("sata1"), //  9
	DISK_ERR_LED("sata1"), // 10
```
:)

Reply:
Great catch!

I was actually planning on tweaking some clang-format settings in a follow-up PR, but I've enabled comment alignments for now (not ideal since it's mixing tabs and spaces, though).